temporalcli/commandsmd/commands.md

```diff
@@ -4,173 +4,390 @@ Commands for the Temporal CLI.
+### temporal batch describe: Show Batch Job progress
 
-The temporal batch describe command shows the progress of an ongoing Batch Job.
+Show the progress of an ongoing Batch Job. Pass a valid Job ID
+to return the job's information:
 
-`temporal batch describe --job-id=MyJobId`
+```
+temporal batch describe --job-id=YourJobId
+```
+
+Read more: https://docs.temporal.io/cli/batch
 
 #### Options
 
 * `--job-id` (string) - The Batch Job Id to describe. Required.
 
 ### temporal batch list: List all Batch Jobs
 
-The temporal batch list command returns all Batch Jobs.
-Batch Jobs can be returned for an entire Cluster or a single Namespace.
+Return a list of Batch jobs, for the entire Service or a single Namespace:
 
-`temporal batch list --namespace=MyNamespace`
+
+```
+temporal batch list --namespace=YourNamespace
+```
+
+Read more: https://docs.temporal.io/cli/batch#list
 
 #### Options
 
-* `--limit` (int) - Limit the number of items to print.
+* `--limit` (int) - Show only the first `N` Batch Jobs in the list.
 
 ### temporal batch terminate: Terminate a Batch Job
 
-The temporal batch terminate command terminates a Batch Job with the provided Job Id.
-For future reference, provide a reason for terminating the Batch Job.
+Terminate the Batch job with the provided Job Id. You must provide
+a reason motivating the termination, which is stored with the Service
+for later reference.
+
+```
+temporal batch terminate --job-id=YourJobId --reason=YourTerminationReason
+```
 
-`temporal batch terminate --job-id=MyJobId --reason=JobReason`
+Read more: https://docs.temporal.io/cli/batch#terminate
 
 #### Options
 
 * `--job-id` (string) - The Batch Job Id to terminate. Required.
 * `--reason` (string) - Reason for terminating the Batch Job. Required.
 
-### temporal env: Manage environments.
+### temporal env: Manage environments
+Environments help you create and manage grouped key-value presets for Options.
+They provide easy set-up for separate environments, such as "dev" and 
+"prod" work.
 
-Use the '--env <env name>' option with other commands to point the CLI at a different Temporal Server instance. If --env
-is not passed, the 'default' environment is used.
+For example, you might set an endpoint preset for the `--address` option.
+You won't have to enter it each time you use the Temporal CLI utility. Env 
+presets let you configure distinct setups for Temporal Services and Namespaces.
 
-### temporal env delete: Delete an environment or environment property.
+Environment names compartmentalize each key-value store. Use the environment
+name as part of your assignment, as in this example, which configures the 
+"prod" environment:
 
-`temporal env delete --env environment [-k property]`
+```
+temporal env set prod.namespace production.f45a2  
+temporal env set prod.address production.f45a2.tmprl.cloud:7233  
+temporal env set prod.tls-cert-path /temporal/certs/prod.pem  
+temporal env set prod.tls-key-path /temporal/certs/prod.key
```

Comment:
```suggestion
temporal env set --env prod --key namespace --value production.f45a2  
temporal env set --env prod --key address --value production.f45a2.tmprl.cloud:7233  
temporal env set --env prod --key tls-cert-path --value /temporal/certs/prod.pem  
temporal env set --env prod --key tls-key-path --value /temporal/certs/prod.key
```

Reply:
Okay, let's discuss this weirdness too. Not against it, just want to make sure that we're on the same page so I add correctly.

Reply:
Sure, so the old `env.key` notation is now deprecated and will eventually be removed (long after 1.0 is released, to give people time to adapt) because it was somewhat confusing. But I'm happy to answer any questions!

Reply:
It's not widely used. At this point, I don't even know who owns those pages or where a ticket would be filed:

```
docs% rgrep -l mdx "prod\."
./docs/cli/env.mdx
./docs/cli/index.mdx
docs% rgrep -l md "prod\."
./node_modules/async/README.md
```